tests/AbstractTestCase.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Psr\Log\Util\Tests;
+
+if (class_exists('PHPUnit\Framework\TestCase')) {
+    class AbstractTestCase extends \PHPUnit\Framework\TestCase {}
+}
+elseif (class_exists('PHPUnit_Framework_TestCase')) {
```

Comment:
Code style should fail here?

Reply:
It should? What gives?

In any case, it's not failing here because PHPUnit is failing before that.